src/Sarif.Multitool.Library/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -15,6 +15,14 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
 {
+    [Flags]
+    public enum RuleKinds
```

Comment:
>RuleKinds [](http://example.com/codeflow?start=16&length=9)

I see we have rules name like:
SARIF2015
GHAS1003
GH1006
ADO1003

the first one SARIF is apply to all,
looks like we are missing GH here.
Should user able to enable any number of the: GH, GHAS,ADO checklist? or just one of them.
e.g. I want to scan once with 3 of them enabled: SARIF+GH+GHAS but not the ADO rules.

Reply:
Yes I think that is how it should work.

Reply:
BTW, I still have cleanup to do wrt naming.